### DISCUSSION TITLE
How can I re-enable the renderer via command line

### DISCUSSION THREAD

**Discussion by Ju1-js** [rocksdanister/lively#1324] [Q&A] [answered]
Is there a command to enable and disable the wallpaper renderer via the command line while ignoring the pause rules?

**Comment by rocksdanister** [comment, marked as answer]
Not possible, pause rules take affect at the very end.

**Reply by Ju1-js** [reply]
How does Lively pause the rendering process, is there a way that I can write another program to pause and unpause the chromium rendering process?

**Reply by Ju1-js** [reply]
Or what file does the pause/unpausing so I can poke around and see what I can do?

**Reply by rocksdanister** [reply]
>How does Lively pause the rendering process, is there a way that I can write another program to pause and unpause the chromium rendering process?

Just turn off pausing in Lively settings and handle it via commandline control:
https://github.com/rocksdanister/lively/wiki/Command-Line-Controls#wallpaper-playback

Do NOT recommend messing with the wallpaper player process from outside Lively.
